docs/nodes/nodes-library/nodes/Medium/README.md

```diff
@@ -0,0 +1,43 @@
+---
+permalink: /nodes/n8n-nodes-base.medium
+---
+
+# Medium
+
+[Medium](https://www.medium.com/) is an online publishing platform and home to a diverse array of stories, ideas, and perspectives. It empowers writers to share their work and ideas with the readers.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Medium/README.md).
+:::
+
+## Basic Operations
+
+- Post
+	- Create a post
+- Publication
+	- Get all publications
+
+
+## Example Usage
+
+This workflow allows you to post an article to a publications on Medium. You can also find the [workflow](https://n8n.io/workflows/594) on the website. This example usage workflow uses the following two nodes.
+
+- [Start](../../core-nodes/Start/README.md)
+- [Medium]()
+
+The final workflow should look like the following image.
+
+![A workflow with the Medium node](./workflow.png)
+
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### 2. Medium node
+
+1. First of all, you'll have to enter credentials for the Medium node. You can find out how to do that [here](../../../credentials/Medium/README.md).
+2. Select the publication from the ***Publication ID*** dropdown list.
```

Comment:
There's a step missing here, the user also had the toggle the ***Publication*** field to true.

Reply:
The workflow already has the configured. When the user copy and paste the workflow in the editor, it is already set true for them.

Reply:
The instructions assume that the reader is setting up the nodes and config options from scratch. The workflow is merely in case the reader wants to skip setting things up from scratch and wants to fast forward.